dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops

```diff
@@ -356,7 +354,8 @@ ERROR_BUILDING_SIGNED_JWE=CWWKS6060E: The [{0}] JWT builder cannot create a JSON
 ERROR_BUILDING_SIGNED_JWE.explanation=The JWT builder encountered an error while creating the JWE token. Check the error that is included in the message for more information.
 ERROR_BUILDING_SIGNED_JWE.useraction=Check the error that is included in the message for more information.
 
+# Do not translate "JwtBuilder"
 # 0=Builder config ID, 1=keyManagementKeyAlias value, 2=Trust store config ID
 KEY_MANAGEMENT_KEY_NOT_FOUND=CWWKS6061E: The [{0}] JWT builder cannot create a JSON Web Encryption (JWE) token because a key management key cannot be found. They key management key alias is [{1}], and the configured trust store is [{2}].
-KEY_MANAGEMENT_KEY_NOT_FOUND.explanation=A key management key is a key that is used to encrypt the Content Encryption Key of a JWE token. A key must be specified so the JWT builder can create JWE tokens.
-KEY_MANAGEMENT_KEY_NOT_FOUND.useraction=Verify that a key with the alias that is specified in the message exists in the trust store that is specified in the message.
+KEY_MANAGEMENT_KEY_NOT_FOUND.explanation=A key management key is a key that is used to encrypt the Content Encryption Key of a JWE token. A key must be specified so the JWT builder can create JWE tokens. A key can be specified by using the configuration attributes that are specified in the message or through the JwtBuilder API.
```

Comment:
`A key must be specified so the JWT builder can create JWE tokens.`

In this sentence, does "key" refer to the key management key or the CEK? I think it means the key management key. If so, this sentence and the one that follows should specify. "key management key":

` A key management key must be specified so the JWT builder can create JWE tokens. A key management key can be specified by using the key alias and configured trust store values that are specified in the message or by using the JwtBuilder API.`

Reply:
Good suggestion - just pushed a new commit with your suggested changes.